journal/src/test/java/io/camunda/zeebe/journal/file/SegmentedJournalTest.java

```diff
@@ -51,10 +53,19 @@ class SegmentedJournalTest {
   private final BufferWriter recordDataWriter = new DirectBufferWriter().wrap(data);
   private final int entrySize = getSerializedSize(data);
 
+  private SegmentedJournal journal;
+
+  @AfterEach
+  void teardown() {
+    if (journal != null) {
```

Comment:
:upside_down_face: I always use Agrona's `CloseHelper.quietClose` since it deals with nulls, quiets any errors if I don't care, etc. Also supports collections via `CloseHelper.quietCloseAll`.